sklearn/pipeline.py

```diff
@@ -787,6 +787,23 @@ def make_union(*transformers, **kwargs):
     and does not permit, naming the transformers. Instead, they will be given
     names automatically based on their types. It also does not allow weighting.
 
+    Parameters
+    ----------
+    transformer_list : list of (string, transformer) tuples
+        List of transformer objects to be applied to the data. The first
+        half of each tuple is the name of the transformer.
+
+    n_jobs : int, optional
+        Number of jobs to run in parallel (default 1).
+
+    transformer_weights : dict, optional
```

Comment:
Should probably leave this out, given that transformer names are automatically assigned, so it only really adds confusion.